### PR TITLE
Italian translation updates and additions

### PR DESCRIPTION
**Pull request by marchitelli**

### CODE REVIEW

File: plugins/importexport/crossref/locale/it_IT/locale.xml [hunk 1]
```diff
@@ -4,32 +4,45 @@
 <!--
   * plugins/importexport/crossref/locale/it_IT/locale.xml
   *
-  * Copyright (c) 2014-2015 Simon Fraser University Library
+  * Copyright (c) 2013-2015 Simon Fraser University Library
```
Comment (mtub) [inline plugins/importexport/crossref/locale/it_IT/locale.xml]: should still be 2014 - see en_US version of the file

**Comment by mtub** [comment]
Hi @marchitelli , thanks for contributing. This was intended for OJS's master branch, right? I'm asking because some of the new translations are still missing in the ojs-dev-2_4 branch.

File: plugins/importexport/crossref/locale/it_IT/locale.xml (hunk 1 above)
Comment (marchitelli) [inline plugins/importexport/crossref/locale/it_IT/locale.xml]: Corrected: thanks for pinting it out

**Comment by mtub** [comment]
Thanks @marchitelli , will merge now.

**Pull request merged**